tests/platform_tests/test_link_down.py

```diff
@@ -190,9 +209,16 @@ def test_link_status_on_host_reboot(duthosts, localhost, enum_frontend_dut_hostn
     # After reboot, immediately check for links 'down' status
     link_status_on_host(duthost, localhost, fanouts_and_ports, up=False)
 
-    time.sleep(MAX_TIME_TO_REBOOT)
+    # Wait for ssh port to open up on the DUT
+    wait_for_startup(duthost, localhost, 0, MAX_TIME_TO_REBOOT)
```

Comment:
same here.
`wait_for_startup` needs to be before `link_status_on_host`

Reply:
To keep the intent of test case intact we should not move wait_for_startup before link_status_on_host. Besides, link_status_on_host is not querying anything on duthost. It is checking link status on the fanouts